twoliter/src/cmd/build.rs

```diff
@@ -13,18 +13,84 @@ use tempfile::TempDir;
 #[derive(Debug, Parser)]
 pub(crate) enum BuildCommand {
     Clean(BuildClean),
+    Kit(BuildKit),
     Variant(BuildVariant),
 }
 
 impl BuildCommand {
     pub(crate) async fn run(self) -> Result<()> {
         match self {
             BuildCommand::Clean(command) => command.run().await,
+            BuildCommand::Kit(command) => command.run().await,
             BuildCommand::Variant(command) => command.run().await,
         }
     }
 }
 
+/// Build a Bottlerocket variant image.
+#[derive(Debug, Parser)]
+pub(crate) struct BuildKit {
+    /// Path to Twoliter.toml. Will search for Twoliter.toml when absent.
+    #[clap(long = "project-path")]
+    project_path: Option<PathBuf>,
+
+    /// The architecture to build for.
+    #[clap(long = "arch", default_value = "x86_64")]
+    arch: String,
+
+    /// The name of the kit to build.
+    kit: String,
+
+    /// The URL to the lookaside cache where sources are stored to avoid pulling them from upstream.
+    /// Defaults to https://cache.bottlerocket.aws
+    lookaside_cache: Option<String>,
```

Comment:
This feels like a `Twoliter.toml` sort of concern - letting developers specify a list of allowed caches, or at least set the cache URL globally for their project.

Reply:
Will do #254